src/forms/pause-dispense-form.workspace.tsx

```diff
@@ -168,16 +175,16 @@ const PauseDispenseForm: React.FC<PauseDispenseFormProps> = ({
             }}
           />
         </section>
-        <section className={styles.buttonGroup}>
-          <Button disabled={isSubmitting} onClick={() => closeWorkspace()} kind="secondary">
-            {t('cancel', 'Cancel')}
-          </Button>
-          <Button disabled={!isValid || isSubmitting} onClick={handleSubmit}>
-            {t(mode === 'enter' ? 'pause' : 'saveChanges', mode === 'enter' ? 'Pause' : 'Save changes')}
-          </Button>
-        </section>
       </div>
-    </div>
+      <section className={styles.buttonGroup}>
+        <Button disabled={isSubmitting} onClick={() => closeWorkspace()} kind="secondary">
+          {t('cancel', 'Cancel')}
+        </Button>
+        <Button disabled={!isValid || isSubmitting} onClick={handleSubmit}>
+          {t(mode === 'enter' ? 'pause' : 'saveChanges', mode === 'enter' ? 'Pause' : 'Save changes')}
```

Comment:
I don't think this gets parsed by the parser correctly.

```suggestion
          {// t('pause', 'Pause')}
          {// t('saveChanges', 'Save changes')}
          {t(mode === 'enter' ? 'pause' : 'saveChanges'')}
```